scripts/convert_freqs.py

```diff
@@ -0,0 +1,79 @@
+import argparse

```

Comment:
This script is getting quite a few steps removed from the original corpus now. It might be better to convert this to a script which reads a large corpus and creates the vocabularies directly, rather than us having created this intermediate file with the word/doc counts in it, and then having this one generate a vocabulary file which is not substantially different apart from how it is filtered.

Reply:
I never really read this code, just copied it around, so you're probably correct

Reply:
#300